### ISSUE TITLE
[Bug] Background Image animation is lagging in middle

### ISSUE THREAD

**Issue by Vikasx08** [Vikasx08/Square-Space#1]
**Bug Description**
Background Image of landing screen is lagging in middle.

**To Reproduce**
1. Open Square Space code on desktop
2. Wait for background images to be entered from left
3. See the lagging while background image animation

**Expected Behaviour**
Animation of background image should not be lagging

**Screen Recording of Error**

https://github.com/Vikasx08/Square-Space/assets/90097260/b77b4123-f30c-47b0-a8e5-33adeab5e888

**Comment by TheVivekYadav** [comment]
**Subject: Re: Issue with Lagging Background Image Animation on Landing Screen**

Hello **Vikasx08**,

I hope this message finds you well. I came across the bug report regarding the lagging background image animation on the landing screen. I forked the code and tested it on my system, but I couldn't replicate the issue you mentioned. **The animation appears to be running smoothly without any lag.**

Since the problem seems to be specific to your environment, there could be a few factors contributing to the lag. Here are some suggestions that might help you investigate further:

**Resource Usage**: Monitor your system's resource usage while running the application. Other background processes might be consuming resources and affecting the animation's performance.

**Network Speed**: Though unlikely, slow internet connectivity could potentially cause assets to load slowly, affecting the animation.

If you require further assistance or encounter any other issues, I'm here to help. **Feel free** to ask, and I'll do my best to contribute to the resolution.

Best regards,
**Vivek Yadav**
